ethereum/sidecar/cli/flags.go

```diff
@@ -12,6 +13,8 @@ const (
 	FlagServerNetwork             = "ethereum-sidecar.server.network"
 	FlagServerBatchSize           = "ethereum-sidecar.server.batch-size"
 	FlagServerRequestsPerMinute   = "ethereum-sidecar.server.requests-per-minute"
+	FlagBridgeOutServerAddress    = "ethereum-sidecar.bridge-out.server-address"
```

Comment:
"Bridge out server" sounds very cryptic. Also, we now have too much clients and servers within the `sidecar` package. What do you think about naming this flag `ethereum-sidecar.assets-unlocked-endpoint`. This way we will be more specific. This will be also consistent with Connect sidecar where they use `market-map-endpoint` which will be easier to explain to operators.

If we agree about that, let's change the entire nomenclature accordingly, so, for example:
- `bridge_out_client.go` becomes just `assets_unlocked.go`
- `BridgeOutGrpcClient` becomes `AssetsUnlockedEndpoint`
- `bridgeOutLookBackPeriod` becomes `AssetsUnlockedLookbackPeriod`

Reply:
Done in 389206bf39397eedfe2187cf18c45c15eeca13fb, 695e7e99bbe88871b97a2e31a7858226ec5185fa, ee74efe8e72f4d7c35b1be9967356a481297ec03.